hedera-node/hedera-network-service-impl/src/test/java/com/hedera/node/app/service/network/impl/test/serdes/MonoRunningHashesAdapterSerdesTest.java

```diff
@@ -16,60 +16,47 @@
 
 package com.hedera.node.app.service.network.impl.test.serdes;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 import com.hedera.node.app.service.mono.stream.RecordsRunningHashLeaf;
 import com.hedera.node.app.service.network.impl.serdes.MonoRunningHashesAdapterCodec;
-import com.hedera.pbj.runtime.io.ReadableSequentialData;
-import com.hedera.pbj.runtime.io.WritableSequentialData;
+import com.hedera.pbj.runtime.io.stream.ReadableStreamingData;
+import com.hedera.pbj.runtime.io.stream.WritableStreamingData;
 import com.swirlds.common.constructable.ClassConstructorPair;
 import com.swirlds.common.constructable.ConstructableRegistry;
 import com.swirlds.common.constructable.ConstructableRegistryException;
 import com.swirlds.common.crypto.Hash;
 import com.swirlds.common.crypto.RunningHash;
 import com.swirlds.common.io.streams.SerializableDataInputStream;
-import com.swirlds.common.io.streams.SerializableDataOutputStream;
-import org.junit.jupiter.api.Test;
-import org.mockito.Mock;
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.Test;
 
 class MonoRunningHashesAdapterSerdesTest {
     private static final RecordsRunningHashLeaf SOME_HASHES = new RecordsRunningHashLeaf(
             new RunningHash(new Hash("abcdefabcdefabcdefabcdefabcdefabcdefabcdefabcdef".getBytes())));
 
-    @Mock
-    private ReadableSequentialData input;
-
-    @Mock
-    private WritableSequentialData output;
-
     final MonoRunningHashesAdapterCodec subject = new MonoRunningHashesAdapterCodec();
 
     @Test
     void doesntSupportUnnecessary() {
         assertThrows(UnsupportedOperationException.class, () -> subject.measureRecord(SOME_HASHES));
-        assertThrows(UnsupportedOperationException.class, () -> subject.measure(input));
-        assertThrows(UnsupportedOperationException.class, () -> subject.fastEquals(SOME_HASHES, input));
+        assertThrows(UnsupportedOperationException.class, () -> subject.measure(null));
```

Comment:
This also changes the effect.  If the intent is to ensure these methods always throw, then perhaps calling with both null and non-null inputs would be advisable.

Reply:
I fixed it thank you